pkgs/by-name/hi/highctidh/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  python3Packages,
+  fetchgit,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "highctidh";
+  version = "1.0.2024060500";
+  pyproject = true;
+
+  src = fetchgit {
+    url = "https://codeberg.org/vula/highctidh";
+    rev = "v${version}";
+    hash = "sha256-TyD5KzUz89RBxsSZeJYOkIzD29DF0BjizpMnsTpFOHI=";
+  };
+
+  sourceRoot = "${src.name}/src";
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
```

Comment:
See [contributing guidelines for python](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines) and [handling dependencies](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#handling-dependencies-handling-dependencies):

> The two parameters, `pyproject` and `build-system` are set to avoid the legacy setuptools/distutils build.

```suggestion
  build-system = [
    setuptools
  ];
```